### PR TITLE
docs: Cascader displayRender data

### LINKED ISSUE #33067: Cascader 多选时 displayRender 不生效
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
业务要求，只显示最后一级的tag标签，且数据也是只要最后一级

### What does the proposed API look like?
提供只显示最后一级tag的属性

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

resolve #33067

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |       -    |
| 🇨🇳 Chinese |     -      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/cascader/index.zh-CN.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ cover: https://gw.alipayobjects.com/zos/alicdn/UdS8y8xyZ/Cascader.svg
 | className | 自定义类名 | string | - |  |
 | defaultValue | 默认的选中项 | string\[] \| number\[] | \[] |  |
 | disabled | 禁用 | boolean | false |  |
-| displayRender | 选择后展示的渲染函数 | (label, selectedOptions) => ReactNode | label => label.join(`/`) |  |
+| displayRender | 单选模式下选择后展示的渲染函数，多选请使用 tagRender | (label, selectedOptions) => ReactNode | label => label.join(`/`) |  |
```
Comment (afc163) [inline components/cascader/index.zh-CN.md:32]: 做个兼容透传？

Comment (zombieJ) [reply components/cascader/index.zh-CN.md:32]: `displayRender` 是带有路径信息的，多选本身就没有路径信息数据结构是不一样的。新 feature 就不做兼容了。

Comment (afc163) [reply components/cascader/index.zh-CN.md:32]: 代码里加个 warning。

Comment (afc163) [reply components/cascader/index.zh-CN.md:32]: 云凤蝶看到这个属性又要吐血了。

Comment (zombieJ) [reply components/cascader/index.zh-CN.md:32]: Cascader 相比其他类 Select 组件更有异域风，搞它的时候也很头疼。我感觉 `rc-select` 得抽一个纯 UI 组件出来才好。

**Pull request merged**